dm/tests/load_task/run.sh

```diff
@@ -170,13 +170,94 @@ function test_transfer_two_sources() {
 		"\"taskStatus\": \"Running\"" 4
 }
 
+function stop_task_left_load() {
+	echo "start DM master, workers and sources"
+	run_dm_master $WORK_DIR/master $MASTER_PORT1 $cur/conf/dm-master.toml
+	check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT1
+
+	export GO_FAILPOINTS="github.com/pingcap/tiflow/dm/loader/LoadDataSlowDownByTask=return(\"load_task1\")"
+	run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
+	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT
+	cp $cur/conf/source1.yaml $WORK_DIR/source1.yaml
+	sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker1/relay_log" $WORK_DIR/source1.yaml
+	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
+
+	dmctl_start_task_standalone "$cur/conf/dm-task-standalone.yaml" "--remove-meta"
+
+	export GO_FAILPOINTS=""
+	run_dm_worker $WORK_DIR/worker2 $WORKER2_PORT $cur/conf/dm-worker2.toml
+	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER2_PORT
+
+	# kill worker1, load_task1 will be transferred to worker2, but lack local files
+	ps aux | grep dm-worker1 | awk '{print $2}' | xargs kill || true
+	check_port_offline $WORKER1_PORT 20
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status load_task1" \
+		"different worker in load stage, previous worker: worker1, current worker: worker2" 1
+
+	# now left a load_task KV in etcd
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"stop-task load_task1" \
+		"\"result\": true" 2
+
+	dmctl_start_task_standalone "$cur/conf/dm-task2-standalone.yaml" "--remove-meta"
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status load_task2" \
+		"\"unit\": \"Sync\"" 1
+
+	# after worker1 goes online, although it has unfinished load_task1, but load_task1 is stopped so should not rebound
+	export GO_FAILPOINTS="github.com/pingcap/tiflow/dm/loader/LoadDataSlowDownByTask=return(\"load_task1\")"
```

Comment:
do we need clean the `GO_FAILPOINTS` env after this test done?

Reply:
I still need this task stuck in load unit so following test can use it